src/main/scala/centaur/api/CromwellClient.scala

```diff
@@ -33,6 +35,19 @@ object CromwellClient {
   final implicit val materializer: ActorMaterializer = ActorMaterializer(ActorMaterializerSettings(system))
   final val apiPath = "/api/workflows/v1"
 
+  def decodeResponse(response: HttpResponse): HttpResponse = {
+    val decoder = response.encoding match {
+      case HttpEncodings.gzip ⇒
+        Gzip
+      case HttpEncodings.deflate ⇒
+        Deflate
```

Comment:
@cjllanwarne it should be a negotation between the server and the client, not something cromwell always does. In fact there are spray directives (which sadly cromwell uses) which akka-http did away with specifically because it didn't allow for negotiation